src/OutputWriters/output_construction.jl

```diff
@@ -2,11 +2,11 @@ using Oceananigans.Fields: validate_indices, Reduction, indices
 using Oceananigans.AbstractOperations: AbstractOperation, ComputedField
 using Oceananigans.Grids: default_indices
 
-restrict_to_interior(::Colon, ::Colon, loc, topo, N) = interior_indices(loc, topo, N)
-restrict_to_interior(::Colon, ::Colon, ::Nothing, topo, N) = UnitRange(1, 1)
-restrict_to_interior(target_index::UnitRange, available_index, ::Nothing, topo, N) = UnitRange(1, 1)
+restrict_to_interior(target_index::Colon,     available_index::Colon, loc,       topo, N) = interior_indices(loc, topo, N)
+restrict_to_interior(target_index::Colon,     available_index::Colon, ::Nothing, topo, N) = UnitRange(1, 1)
```

Comment:
Doesn't follow YASGuide but we can make an exception...

Reply:
Damn.. Sorry... I was trying to make things cleaner by having the positional argument names